pkg/util/winutil/iisconfig/apmtags.go

```diff
@@ -0,0 +1,103 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+//go:build windows
+
+// Package iisconfig manages iis configuration
+package iisconfig
+
+/*
+the file datadog.json can be located anywhere; it is path-relative to a .net application
+give the path name, read the json and return it as a map of string/string
+*/
+
+import (
+	"encoding/json"
+	"encoding/xml"
+	"os"
+)
+
+type APMTags struct {
+	DDService string
+	DDEnv     string
+	DDVersion string
+}
+
+// ReadDatadogJSON reads a datadog.json file and returns the APM tags
+func ReadDatadogJSON(datadogJSONPath string) (APMTags, error) {
+	var datadogJSON map[string]string
+	var apmtags APMTags
+
+	file, err := os.Open(datadogJSONPath)
+	if err != nil {
+		return apmtags, err
+	}
+	defer file.Close()
+
+	decoder := json.NewDecoder(file)
+	err = decoder.Decode(&datadogJSON)
+	if err != nil {
+		return apmtags, err
+	}
+	apmtags.DDService = datadogJSON["DD_SERVICE"]
```

Comment:
Since these fields are user editable, should they be sanitized?
I don't know what guarantees Go provides .

Reply:
sanitized in what way?  This should be configured by the customer to be the service value they want to send.
(our tags will be post-processed later to remove special chars, etc.)

Reply:
OK, so if there is no processing of the values (e.g. copy-pasted strings), should be fine.